scripts/package.json

```diff
@@ -196,7 +196,7 @@
     "verdaccio": "^5.31.1",
     "verdaccio-auth-memory": "^10.2.2"
   },
-  "packageManager": "yarn@4.4.0+sha512.91d93b445d9284e7ed52931369bc89a663414e5582d00eea45c67ddc459a2582919eece27c412d6ffd1bd0793ff35399381cb229326b961798ce4f4cc60ddfdb",
```

Comment:
Using the hash is recommended by yarn, but seldom used. This package was the only one using it, so decided to stay with the auto generated output for now.